obspy/core/stream.py

```diff
@@ -1530,13 +1533,14 @@ def trim(self, starttime=None, endtime=None, pad=False,
                     endtime = tr.stats.endtime + delta * tr.stats.delta
             except TypeError:
                 msg = ('starttime and endtime must be UTCDateTime objects '
-                       'or None for this call to Stream.trim()')
+                       'or None for this call to Stream.slice()')
```

Comment:
isn't this supposed to state `trim`? seems to be inside a trim call, no?

Reply:
You are right. I changed the wrong error message.